CRBM-Viz/src/app/Components/loading-spinner/loading-spinner.component.sass

```diff
@@ -0,0 +1,49 @@
+@keyframes lds-eclipse
```

Comment:
Angular material already has a spinner defined. 
Can we use that instead for simplicity and consistancy? 
https://material.angular.io/components/progress-spinner/overviewhttps://material.angular.io/components/progress-spinner/overview